dependabot/full_build_pipeline_for_updated_stages.py

```diff
@@ -239,7 +239,16 @@ def switch_to_branches_from_updated_stages():
                 exit_code = os.system(f"cd {module['name']};git checkout 1.0.x")
 
                 if exit_code != 0:
-                    print(f"Failed to switch to branch 'v1.0.x' from last updated commit id for " +
+                    print(f"Failed to switch to branch '1.0.x' from last updated commit id for " +
+                          f"{module['name']}")
+                    sys.exit(1)
+                continue
+            elif module['name'] == "module-ballerina-websubhub":
```

Comment:
For `module-ballerina-websub` also we hv a branch for patches(2201.0.x). Shall we include that also ?

Reply:
As per the offline discussion will proceed with this way only for the failed modules. Other modules should always use the previously released tags.

Reply:
@ayeshLK Since we do this only for failing modules (due to test changes) which cannot be released due to no changes, we should use the last released stage.